$OpenBSD: patch-tools_build_src_engine_jam_h,v 1.5 2022/01/06 08:15:52 rsadowski Exp $

Do not unconditionally define unix on OpenBSD.
dcffeb632efc17709d1b6ba491c00b520d398e5a

Index: tools/build/src/engine/jam.h
--- tools/build/src/engine/jam.h.orig
+++ tools/build/src/engine/jam.h
@@ -328,7 +328,9 @@
 #ifdef __OpenBSD__
     #define OSMINOR "OS=OPENBSD"
     #define OS_OPENBSD
+# if !defined(unix)
     #define unix
+# endif
 #endif
 #if defined (__FreeBSD_kernel__) && !defined(__FreeBSD__)
     #define OSMINOR "OS=KFREEBSD"
